### PR TITLE
[7152] Jd 2023 03 more translations

### PR DESCRIPTION
**Pull request by goapunk**
depends on #421

### CODE REVIEW

**Comment by khamui** [comment]
@goapunk uppsala...needs a rebase and some conflict solving :)

**Comment by khamui** [comment]
not merged yet, due to an error.

Bildschirm­foto 2023-03-09 um 11 22 50

**Comment by khamui** [comment]
@fuzzylogic2000 @Rineee i had an error (see above) when testing this PR. I added a commit which makes it work, but I think @goapunk solution must somehow worked and is actually the way to go. My "fix" has some downsides for instance multi line text is looking off...

**Comment by fuzzylogic2000** [comment]
@khamui YEs, this is very weird! I also had a look and got it working for some of the <Trans> elements, but not for all of them and it does not make any sense. Sometimes even ones that should be straightforward like in Phases does not work. So, I don't know what to do here. :( 
Maybe merge with the removal of the elements and then trying again later?

File: src/containers/Comments/Comment.js [hunk 1]
```diff
@@ -61,16 +63,15 @@ export const Comment = (props) => {
   const getCommentDeleteModalItems = (commentInstance) => {
     return [
       {
-        // space is to center the text
-        title: '   This comment will be deleted.\nThis action cannot be undone.',
+        title: [t('This comment will be deleted.'), t('This action cannot be undone.')],
```
Comment (goapunk) [inline src/containers/Comments/Comment.js:66]: removed the whitespaces as it doesn't work for translated strings anymore (see comment above)

**Comment by goapunk** [comment]
@khamui @fuzzylogic2000 I fixed some issues and made a small change to the modal, this is ready for review again

File: src/containers/Comments/Comment.js (hunk 1 above)
Comment (fuzzylogic2000) [reply src/containers/Comments/Comment.js:66]: Cool! This is much better and on android it's still centered. :)

**Review by fuzzylogic2000 — APPROVED** [review]
Cool, works for me (on android).
@khamui I suppose you also want to test iOS?

File: src/components/ModalItem.js [hunk 2]
```diff
@@ -37,9 +37,11 @@ export const ModalItem = (props) => {
       containerStyle={containerStyles}
     >
       <ListItem.Content style={styles.modalContent}>
-        <ListItem.Title style={listItemStyles}>
-          {props.item.title}
-        </ListItem.Title>
+        {titleList.map((t,i) => (
```
[the remark this answers is not in the document]
Comment (khamui) [reply src/components/ModalItem.js:40]: nice!

File: src/containers/Auth/LoginScreen.js [hunk 3]
```diff
@@ -74,17 +75,17 @@ export const LoginScreen = () => {
         >
           <View style={styles.container}>
             <TextSourceSans style={styles.title}>
-              Login
+              {t('Login')}
             </TextSourceSans>
-            <Trans>
-              <TextSourceSans style={styles.registerText}>
+            <TextSourceSans style={styles.registerText}>
+              <Trans t={t}>
```
Comment (khamui) [inline src/containers/Auth/LoginScreen.js:81]: is this prop needed?

**Review by khamui — APPROVED** [review]
really cool! working fine in iOS!

Just one question to the `<Trans...` tag. Feel free to merge!

File: src/containers/Auth/LoginScreen.js (hunk 3 above)
Comment (goapunk) [reply src/containers/Auth/LoginScreen.js:81]: I  think it's needed to update the view if the language changes, if you just use <trans> it wouldn't. At this point it's probably more of a theoretical problem as you can't really change the language in that view

**Pull request merged**